contracts/0.4.24/Lido.sol

```diff
@@ -241,34 +276,24 @@ contract Lido is ILido, IsContract, StETH, AragonApp {
     }
 
     /**
-      * @notice Set authorized oracle contract address to `_oracle`
-      * @dev Contract specified here is allowed to make periodical updates of beacon states
-      * by calling handleOracleReport.
-      * @param _oracle oracle contract
-      */
-    function setOracle(address _oracle) external auth(SET_ORACLE) {
-        _setOracle(_oracle);
-        emit OracleSet(_oracle);
-    }
-
-    /**
-      * @notice Set treasury contract address to `_treasury`
-      * @dev Contract specified here is used to accumulate the protocol treasury fee.
-      * @param _treasury contract which accumulates treasury fee.
-      */
-    function setTreasury(address _treasury) external auth(SET_TREASURY) {
-        _setTreasury(_treasury);
-        emit TreasurySet(_treasury);
-    }
-
-    /**
-      * @notice Set insuranceFund contract address to `_insuranceFund`
-      * @dev Contract specified here is used to accumulate the protocol insurance fee.
-      * @param _insuranceFund contract which accumulates insurance fee.
-      */
-    function setInsuranceFund(address _insuranceFund) external auth(SET_INSURANCE_FUND) {
-        _setInsuranceFund(_insuranceFund);
-        emit InsuranceFundSet(_insuranceFund);
+    * @notice Set Lido DAO contracts (oracle, treasury, insurance fund).
+    *
+    * @dev Oracle contract specified here is allowed to make
+    * periodical updates of beacon states
+    * by calling pushBeacon. Treasury contract specified here is used
+    * to accumulate the protocol treasury fee.Insurance fund contract
+    * specified here is used to accumulate the protocol insurance fee.
+    *
+    * @param _oracle oracle contract
+    * @param _treasury treasury contract which accumulates treasury fee
+    * @param _insuranceFund insurance fund contract which accumulates insurance fee
+    */
+    function setDAOContracts(
+        address _oracle,
+        address _treasury,
+        address _insuranceFund
+    ) external auth(MANAGE_DAO_CONTRACTS_ROLE) {
```

Comment:
Actually I like how `MANAGE_DAO_CONTRACTS_ROLE` sounds. 
Just want to share a thought. 
That "DAO contracts" may be comprehended as "contracts related to DAO governance"
Maybe "protocol contracts" would be a bit closer

Reply:
Yep, sounds great, ty for the input!